Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml

```
@@ -88,7 +88,7 @@
     price: 3000
 
 - type: entity
-  parent: [ClothingShoesBootsMagBase, BaseJetpack, BaseSyndicateContraband]
+  parent: [ClothingShoesBootsMagBase, BaseJetpack, BaseSyndicateContraband, ClothingShoesMilitaryBase] #FH edit
   id: ClothingShoesBootsMagSyndie
   name: blood-red magboots
   description: Reverse-engineered magnetic boots that have a heavy magnetic pull and integrated thrusters. It can hold 0.75 L of gas.
```

Adding a little bit to description so people know it now has a knife/small arms slot.
```suggestion
  description: Reverse-engineered magnetic boots that have a heavy magnetic pull and integrated thrusters. It can hold 0.75 L of gas. Has also been equipped with knife/small arms storage for Operatives on the go. #Far Horizons
```

---

omfg i actually did this in dev but forgot to save the changes </3, thanks for catching